homeassistant/components/sia/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""The sia integration."""
+import asyncio
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN, PLATFORMS
+from .hub import SIAHub
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the sia component."""
+    hass.data[DOMAIN] = {}
```

Comment:
I would consider using `.setdefault()` since I'm not sure if this can get called somehow after there's already data there.

Reply:
tried `hass.data[DOMAIN].setdefault()` but that gave an error, or did you mean another way?

Reply:
The right syntax is `hass.data.setdefault(DOMAIN, {})`

Reply:
Perfect, thanks, prepping a commit now.